heartbeat/tests/system/test_monitor.py

```diff
@@ -75,6 +75,43 @@ def test_tcp(self, port, status):
             raise SkipTest
         self.assert_fields_are_documented(output[0])
 
+    def test_config_reload(self):
+        """
```

Comment:
Two things I would like to see also in this or an additional test:

* Removing of monitors and making sure the monitor is stopped and does not report anymore
* Mix with the poll file and reloading. Will this work?

An other option that we could do is allow the pollfile only for static configs which are not reloaded but I assume that would actually make our code more complicated.